### DISCUSSION TITLE
Starting Corporation Draft is being deleted (edited from original title)

### DISCUSSION THREAD

**Discussion by DERTAZ** [bafolts/terraforming-mars#5597] [Ideas]
Some time ago there was an option: corporate draft, I used it often. This option has been missing for some time. Probably changed by amendment #13209, why this option disappeared and is there a chance that it will come back.

**Comment by kberg** [comment]
Corporation Draft was disabled on March 24: [Discord ref](https://discord.com/channels/737945098695999559/763150891498733599/1088854947354918954)

and it will be deleted in the next month.

I asked for feedback on January 7 [Discord ref](https://discord.com/channels/737945098695999559/738006783955697704/1061340224455716906)

Almost every bit of feedback on the corporate draft was that it wasn't used because it wasn't a proper draft. There are many forms of draft,but this one was considered unbalanced, and also different from the other drafts as implemented.

For example:
>  IMHO, it's very poorly implemented. In a 3 player game, if you choose 3 corps per player and corporation draft, all players see the 9 corporations available. Player 1 picks a corp from 9 available. Player 2 picks a corp from 8. Player 3 picks a corp from 7, and etc. However, in the online implementation, unless you're playing live or watching constantly, player 3 will only know that 2 corps were picked, and not which player picked which corp. Also, by the time you get to the 9th card, it's likely you'll have no idea who picked what. And then the whole tableau disappears when the card draft starts, so unless you took notes or a screenshot, you won't know which corps were available. It's a classic case of keeping trackable information hidden.

... and later on

>  we like the idea of the corporation draft, if it was implemented the same as the other draffts

[Discord ref](https://discord.com/channels/737945098695999559/787457316009803786/1080501485630128188)

A branch with this kind of corporation draft is still available on a Github repository branch, see https://github.com/terraforming-mars/terraforming-mars/tree/corporate-draft

**Reply by DERTAZ** [reply]
It's a pity.

My wife and I play in such a way that in each season we play, for example, twice with each corporation. And this draft option helped us a lot at the end of the season to be able to choose the corporations for each person - which can still play in a given season.

**Reply by kberg** [reply]
> It's a pity.
> 
> My wife and I play in such a way that in each season we play, for example, twice with each corporation. And this draft option helped us a lot at the end of the season to be able to choose the corporations for each person - which can still play in a given season.

There's a better form of corporate draft out there that we should replace it with.

Also, if you and your wife are playing, perhaps you can host a game yourself locally (as in, at home, shared on your wifi network.) It's not trivial without the technical experience, but it's certainly doable.
